### DISCUSSION TITLE
Не могу найти статистику по жанрам

### DISCUSSION THREAD

**Discussion by F000x** [Chimildic/YaMuTools#26] [Вопрос/ответ] [answered]
Подскажите, пожалуйста, а где можно увидеть статистику по жанрам? Не нашел такого, а в статье на tjournal читал, что есть такая функция

**Comment by Chimildic** [comment, marked as answer]
Функция статистики была удалена и пока не возвращалась.

**Comment by KisoidPoh095** [comment]
> статистику по жанрам?

Некоторые работники музыкальной редакции ЯМузыки находятся "на своей волне", так что толку от такой статистики будет немного.

Вот, например: https://music.yandex.ru/album/7765519
Сборник называется "Jazz Lounge". Вообще думать не надо! Но нет, запихнули в "танцевальную музыку"...

Самый простой из известных мне вариантов: завести аккаунт в Ласт.ФМ и смотреть в еженедельном отчёте "Статистику по тегам".
